src/settings/LibraryHours.js

```diff
@@ -62,10 +62,9 @@ class LibraryHours extends React.Component {
         renderedCloneSettings = <CloneSettings {...this.props} onToggle={that.onChildToggle()} />;
       }
       const sortedList = sortBy((this.props.resources.entries || {}).records || [], ['name']);
-      if (sortedList !== undefined && sortedList !== null) {
-        for (let i = 0; i < sortedList.length; i++) {
-          sortedList[i].allEntries = sortBy((this.props.resources.entries || {}).records || [], ['name']);
-        }
+
+      for (let i = 0; i < sortedList.length; i++) {
+        sortedList[i].allEntries = sortedList;
```

Comment:
I am wondering do we need to store `allEntries` field at all in every arr element?

Reply:
in current realization we need it, or page with 'add exeptions' will crash.